src/pkg/errs/core/core.go

```diff
@@ -61,6 +61,11 @@ var (
 	// about what it sounds like: we tried to look for a backup by ID, but the
 	// storage layer couldn't find anything for that ID.
 	ErrBackupNotFound = &Err{msg: "backup not found"}
+	// occurs when creation of an entity (usually by restful POST or PUT) errors
+	// because some other entity already already exists with a conflicting identifier.
+	// The identifier is not always the id.  For example: duplicate filenames
+	// in the same directory will cause conflicts, even with different IDs.
+	ErrConflictAlreadyExists = &Err{msg: "conflict: already exists"}
```

Comment:
I'm kind of on the fence with the error naming. At least for me reading it the first time it sounded like the conflict already existed, not the item that created the conflict. Could we maybe rename it to `ErrItemAlreadyExists` or `ErrItemConflict` (or something else if there's a better idea)?

Reply:
sounds good!